ui/user/src/lib/components/admin/CatalogServerForm.svelte

```diff
@@ -946,7 +946,14 @@
 			<span class="text-sm font-medium text-red-500">Fill out all required fields</span>
 		{/if}
 		<button class="button flex items-center gap-1" onclick={() => onCancel?.()}> Cancel </button>
-		<button class="button-primary flex items-center gap-1" onclick={handleSubmit}>
+		<button
```

Comment:
This was the simplest fix that made sense to me. Something more complex would be highlighting the "Add MCP Server" section of the `CompositeRuntimeForm` when this button is clicked and no MCP servers are present.